host/sgx/sgxquote.c

```diff
@@ -375,10 +382,23 @@ static void _load_sgx_dcap_ql_impl(void)
             (void**)&_sgx_qe_get_quote_size));
         OE_CHECK(_lookup_function(
             _ql_module, "sgx_qe_get_quote", (void**)&_sgx_qe_get_quote));
+        OE_CHECK(_lookup_function(
+            _ql_module, "sgx_ql_set_path", (void**)&_sgx_ql_set_path));
```

Comment:
"sgx_ql_set_path" only exists in Linux implementation. In Windows, result is going to be OE_FAILURE and dcap_ql will not be unloaded.

Reply:
I can come up with two options.

1. Calling the QE3 `sgx_set_qpl_path` directly.
   It was introduced in both Linux and Windows [in DCAP 1.6](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/DCAP_1.6/QuoteGeneration/quote_wrapper/quote/qe_logic.cpp#L223).
In Linux, as we dlopen() QL(`libsgx_dcap_ql.so`), QE3(`libsgx_qe3_logic.so`) symbols also become available via dlsym().
I'm not sure if Windows LoadLibrary() and GetProcAddress() works the same way.

2. Do not apply this patch in Windows
   Perhaps in Windows something similar to `LD_PRELOAD` is enough to choose a QPL because both Azure DCAP Provider and Intel DCAP Provider use the same name `dcap_quoteprov.dll`.

@yentsanglee I have little experience with Windows development so please tell me what you think.